kolibri/content/api.py

```diff
@@ -57,7 +57,7 @@ def get_ancestor_topics(channel_id=None, content=None, **kwargs):
     :param content: ContentMetadata or str
     :return: QuerySet of ContentMetadata
     """
-    return content.get_ancestors().filter(kind="topic").using(channel_id)
+    return content.get_ancestors().filter(kind__kind="topic").using(channel_id)
```

Comment:
@66eli77 writing a variable name with capitals signals that it's a constant, i.e. that the value shouldn't change at run time. This is a convention. Python doesn't support real constants afaik (or maybe there's some weird implementation).